PWGHF/D2H/Tasks/taskLb.cxx

```diff
@@ -247,7 +425,7 @@ struct HfTaskLbMc {
     for (const auto& particle : mcParticles) {
       if (std::abs(particle.flagMcMatchGen()) == 1 << hf_cand_lb::DecayType::LbToLcPi) {
 
-        auto yParticle = RecoDecay::y(particle.pVector(), o2::constants::physics::MassLambdaB0);
+        auto yParticle = RecoDecay::y(std::array{particle.px(), particle.py(), particle.pz()}, o2::constants::physics::MassLambdaB0);
```

Comment:
Why are you reverting this?

Reply:
Sorry. Did not notice the update. It is now back in.